oe_corporate_blocks.post_update.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Corporate Blocks post updates.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Update corporate footer.
```

Comment:
We need a better description here, such as: 
> Only display "European Commission website" link in footer.

Reply:
Done